test/simulator/moves/ragepowder.js

```diff
@@ -27,15 +28,15 @@ describe('Rage Powder', function () {
 		let hitCount = [0, 0, 0];
 		battle.p1.active[0].damage = function (...args) {
 			hitCount[0]++;
-			return BattleEngine.BattlePokemon.prototype.damage.apply(this, args);
+			return BattlePokemon.prototype.damage.apply(this, args);
```

Comment:
@Slayer95 I'd like to try and keep dependencies explicit. It's not entirely clear (as a newcomer to this code, at least), that the `this` inside of `function (...args)` is going to be of type `BattlePokemon`.